### DISCUSSION TITLE
Are you able to add ExpressVpn

### DISCUSSION THREAD

**Discussion by marcthespot0** [haugene/docker-transmission-openvpn#2226] [Ideas]
It would be great if you could. Thank you for considering!

**Comment by ilike2burnthing** [comment]
Already there - https://github.com/haugene/vpn-configs-contrib/tree/main/openvpn/expressvpn

**Comment by marcthespot0** [comment]
Hi!Thank you for getting back to me so quickly. I want to set this up with unraid.  Not seeing how to get Expressvpn in https://github.com/haugene/docker-transmission-openvpn Any tips would be appreciated! Sent from Mail for Windows From: ilike2burnthingSent: Wednesday, March 16, 2022 7:38 PMTo: haugene/docker-transmission-openvpnCc: marcthespot0; ManualSubject: Re: [haugene/docker-transmission-openvpn] Are you able to add ExpressVpn (Discussion #2226) Already there - https://github.com/haugene/vpn-configs-contrib/tree/main/openvpn/expressvpn—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by ilike2burnthing** [comment]
Documentation - https://haugene.github.io/docker-transmission-openvpn/
